kong/templates/nginx_kong.lua

```diff
@@ -439,11 +439,14 @@ server {
         }
     }
 
+> if legacy_hybrid_protocol == "off" then
```

Comment:
Here we need not to compare to `off`, it is a boolean, just use `if legacy_hybrid_protocol`.

You can refer to `legacy_worker_events`

Reply:
This actually causes a bug. Here's the fix: #9085